Encounter Me.Api/Controllers/TrailController.cs

```diff
@@ -29,34 +29,34 @@ public IActionResult GetTrailById(int id)
         {
             return Ok(_trailRepository.GetTrailById(id));
         }
-        [HttpPost]
-        public IActionResult CreateTrail([FromBody] TrailContainer trail)
-        {
-            if (trail == null)
-                return BadRequest();
+        //[HttpPost]
+        //public IActionResult CreateTrail([FromBody] TrailContainer trail)
+        //{
+        //    if (trail == null)
+        //        return BadRequest();
```

Comment:
Would be nice to know a reason for parts of the code being commented out.

Reply:
This was mentioned in last commit message: `Methods for deletion/update of user data are not necessary as for now we do  not have any business logic that would use them.`